### PR TITLE
Backports for 5.0.0 and update changelog

### PR DESCRIPTION
**Pull request by aaronayres35**
[Targeting maint/5.0.0]

This PR backports merged PRs to the 5.0.0 branch in preparation for the upcoming release.  It also updates the changelog to include these backported PRs.  Only the last commit which updates the changelog needs to be reviewed (the others have been reviewed already).

### CODE REVIEW

**Review by rahulporuri — APPROVED** [review]
LGTM. Can you hold off on doing the 5.0.0 release until after #377 ?

File: CHANGES.rst [hunk 1]
```diff
@@ -32,6 +33,8 @@ Fixes
 Documentation
 -------------
 
+- Contribute examples to etsdemo (#380)
```
Comment (aaronayres35) [inline CHANGES.rst:36]: I wasn't exactly sure the best category to place all of these.  Primarily this one, and 377.  If a reviewers opinion differs, I am happy to move them around

File: CHANGES.rst [hunk 2]
```diff
@@ -22,6 +22,7 @@ Features
 Fixes
 -----
 
+- Make example run from any directory (#377)
```
Comment (aaronayres35) [inline CHANGES.rst]: I ended up moving this to sit with the other recent changes in documentation.  I'm mentally classifying examples as documentation and figured these three related issues should all sit together

**Review by rahulporuri — APPROVED** [review]
still LGTM

**Pull request merged**